sdks/wasm/Makefile

```diff
@@ -155,15 +150,12 @@ endef
 
 $(eval $(call InterpBuildTemplate,debug,wasm-runtime-release,$(EMCC_DEBUG_FLAGS)))
 $(eval $(call InterpBuildTemplate,release,wasm-runtime-release,$(EMCC_RELEASE_FLAGS)))
+$(eval $(call InterpBuildTemplate,release-dynamic,wasm-runtime-release,$(EMCC_RELEASE_DYNAMIC_FLAGS)))
 ifdef ENABLE_WASM_THREADS
```

Comment:
This should be controlled by an ifdef as well. We can define the ifdef in our CI in scripts/ci/run-jenkins.sh so the packages would contain the build.